ios/Exponent/Kernel/AppLoader/CachedResource/EXCachedResource.h

```diff
@@ -8,16 +8,14 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface EXCachedResource : NSObject <EXResourceLoader>
 
+@property (nonatomic, readonly) NSString *resourceName;
```

Comment:
how come we are moving this field from EXManifestResouce?

Reply:
it already exists on `EXCachedResource` and in there we run in through `[EXUtil escapedResourceName:]` https://github.com/expo/expo/blob/c5a18838f760ccb06b4c4c799bcb85d5b9ce005c/ios/Exponent/Kernel/AppLoader/CachedResource/EXCachedResource.m#L31 so this allows `EXManifestResource` to access that property.

We could instead keep the property on `EXManifestResource` and also run it through that selector, but I thought this way was cleaner (the other way, the subclass has a property with the same name as the super but the property is actually different). Let me know if you think that's okay.